05_plot_cancer_type_results.ipynb

```diff
@@ -0,0 +1,1539 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Single-cancer holdout analysis"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib_venn import venn2\n",
+    "import seaborn as sns\n",
+    "\n",
+    "import pancancer_evaluation.config as cfg\n",
+    "import pancancer_evaluation.utilities.analysis_utilities as au"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here, we want to look at experiments where a single cancer type is held out. We want to compare cross-validation results when models are trained only on data from a single cancer type with results when models are trained on data from all cancer types.\n",
```

Comment:
I don't think I'm completely following this part. So for a single-cancer type analysis, you're training on genome data from say cancer type A and you're trying to predict mutation status for BRCA in cancer type A? Or predict in a different cancer type B?

For pancancer is it that you train on genomic data from all cancer types except cancer A and predict mutation status of BRCA on cancer type A?

Reply:
> So for a single-cancer type analysis, you're training on genome data from say cancer type A and you're trying to predict mutation status for BRCA in cancer type A? Or predict in a different cancer type B?

We're always training and testing on the same cancer type (so train on cancer type A and test on cancer type A).

> For pancancer is it that you train on genomic data from all cancer types except cancer A and predict mutation status of BRCA on cancer type A?

In the "pancancer" experiments, we're training on cancer type A + all other cancers, and testing on cancer type A. We haven't tried training on all other cancers and testing on A; that might be a good next step! That experiment could help us better understand where the signal is coming from.